src/components/Skeleton/Col.js

```diff
@@ -10,7 +10,10 @@ const colSizes = {
   '10': 'ten',
   '2': 'two'
 };
-const Col = ({ size, children, className }) => <div className={`${colSizes[size]} columns ${className ? className : ''}`}>{children}</div>;
+const Col = React.forwardRef(({ size, children, className }, ref) => (
+  <div ref={ref} className={`${colSizes[size]} columns ${className ? className : ''}`}>{children}</div>
+));
+
```

Comment:
Miért kell itt a ref? Propokkal nem lehetne megoldani?

Reply:
Azért kellett a ref, mert a MultiSelect komponensbe kellett refet használnom, hogy elrejtsem az options listát, ha a komponensen kívülre kattintanak. 
Ha így használom a refet:
`<Col ref={this.wrapperRef}>
...
</Col>
`

A Col komponens nem fogja megkapni a szülőtől a ref-et, a forwardRef függvény nélkül.